ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -246,3 +246,35 @@ def test_tensorflow_pinv(
         validate_args=False,
         name=None
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=_solve_get_dtype_and_data(),
```

Comment:
This `composite function` does not return `axis`, which causes the frontends tests to fail for `tensordot()`, causing an error: `tensordot() missing 1 required positional argument: 'axes'`.
You have to figure out a way to generate `axis` aswell.
You can use some other composite function or write your own if one doesn't exists, which generates the arguments that are needed.
Refer to the docs for more details. (https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html#examples)